objectRecognitionSource/README.md

```diff
@@ -149,7 +149,8 @@ The Configuration document may look similar to the following example:
           "general": {
              "pollTime": 3000,
              "maxRunningThreads": 5,
-             "maxQueuedTasks": 10
+             "maxQueuedTasks": 10,
+             "suppressNullValues": true
```

Comment:
`suppressEmptyNeuralNetResults` or something.  "null values" is much too general.